micro-ui/web/micro-ui-internals/packages/modules/core/src/components/Home.js

```
@@ -126,22 +143,62 @@
       </div>
-    </div>
+
+      {additionalComponent &&
+        additionalComponent?.length > 0 &&
+        additionalComponent.map((i) => {
+          const Component =
+            typeof i === "string"
+              ? Digit.ComponentRegistryService.getComponent(i)
+              : null;
+          return Component ? (
+            <div className="additional-component-wrapper">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)